src/tabrmd-init.c

```diff
@@ -99,6 +99,12 @@ gmain_data_cleanup (gmain_data_t *data)
     if (data->loop != NULL) {
         main_loop_quit (data->loop);
     }
+    if (data->tpm2 != NULL) {
+        if (data->tpm2->sapi_context != NULL) {
+            g_free(data->tpm2->sapi_context);
```

Comment:
This doesn't seem to follow the same pattern as

https://github.com/tpm2-software/tpm2-abrmd/blob/01a3f6297261085b8ff5b6deb3fbacf14b5e50d6/src/tpm2.c#L89-L92